mlc_llm/utils.py

```diff
@@ -47,7 +47,7 @@ class Quantization:
     ),
 }
 
-supported_model_types = set(["llama", "gpt_neox", "moss", "rwkv"])
+supported_model_types = set(["llama", "gpt_neox", "moss", "rwkv", "gorilla"])
```

Comment:
This line specifies the model architectures we now support. Given Gorilla is a LLaMA-family model, we don’t need to add it here.